pkg/controller/controller_instance_test.go

```diff
@@ -1771,6 +1771,52 @@ func TestReconcileServiceInstanceDeleteFailedProvisionWithoutRequest(t *testing.
 	assertNumEvents(t, events, 0)
 }
 
+// TestReconcileServiceInstanceDeleteFailedProvisionWithoutRequestWithActiveBinding tests
+// that an instance that failed to provision without making a provision request
+// will be finalized even if it has binding(s) and no deprovision request will be sent to the broker.
+func TestReconcileServiceInstanceDeleteFailedProvisionWithoutRequestWithActiveBinding(t *testing.T) {
```

Comment:
"Active" binding seems to imply to me that the binding resource was created at the broker, which isn't the case here. "Inactive" seems to convey the correct intent; the k8s resource was created, but the binding it represents hasn't been created yet.